src/main/java/com/raisetech/todo/controller/ToDoController.java

```diff
@@ -1,24 +1,17 @@
 package com.raisetech.todo.controller;
 
-import com.raisetech.todo.exception.ResourceNotFoundException;
+import com.raisetech.todo.form.ToDoForm;
 import com.raisetech.todo.service.ToDoEntity;
 import com.raisetech.todo.service.ToDoService;
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
 
-import javax.servlet.http.HttpServletRequest;
-import java.time.ZonedDateTime;
 import java.util.List;
-import java.util.Map;
 
 @RestController
 @RequiredArgsConstructor
-public class ToDoController {
+public class ToDoController{
```

Comment:
フォーマッタの設定が悪さしているのですかね？
```suggestion
public class ToDoController {
```

Reply:
もしかしたら無意識で触ってしまったのかもしれません……。気をつけます。
クラス定義開始の波括弧前に空白を追加しました。
d559193f5b0e3a9f64b8d7a9c6917064456c47e4

Reply:
コミットハッシュ貼ってくれていて助かります。